modules/docker-build/variables.tf

```diff
@@ -76,3 +76,9 @@ variable "keep_remotely" {
   type        = bool
   default     = false
 }
+
+variable "platform" {
+  description = "The target architecture platform to build the image for."
+  type        = string
```

Comment:
a bit odd that it wants a string instead of an array of strings, but alas https://github.com/kreuzwerker/terraform-provider-docker/blob/617f2ed44e45d45b07161d3f194c3c324ef0511e/internal/provider/resource_docker_registry_image.go#L315